backend/src/main/java/heartbeat/controller/source/dto/GithubResponse.java

```diff
@@ -1,4 +1,4 @@
-package heartbeat.controller.source.vo;
+package heartbeat.controller.source.dto;
```

Comment:
PLZ help to rename this file name to `GitHubResponse`, and next file `GitHubController`